### PR TITLE
navi: add page

### PR DESCRIPTION
**Pull request by cbr4l0k**
- [x] The page(s) are in the correct platform directories: `linux`.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **navi 2.23.0**

### CODE REVIEW

File: pages/linux/navi.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# navi
+
+> An interactive cheatsheet tool for the command line and application launchers.
+> More information: <https://github.com/denisidoro/navi>.
+
+- Browse through all available cheatsheets:
+
+`navi`
+
+- Browse the cheatsheet for navi itself:
```
Comment (kbdharun) [inline pages/linux/navi.md:10]:
```suggestion
- Browse the cheatsheet for `navi` itself:
```
We generally enclose commands inside back ticks.

File: pages/linux/navi.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`navi fn welcome`
+
+- Print a command from the cheatsheet without executing it:
+
+`navi --print`
+
+- Output shell widget source code. Automatically detects your shell if possible, but can be specified:
```
Comment (kbdharun) [inline pages/linux/navi.md:18]:
```suggestion
- Output shell widget source code (It automatically detects your shell if possible, but can also be specified manually):
```

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**